pkgpanda/build/__init__.py

```diff
@@ -1182,23 +1192,24 @@ def clean():
     # TODO(cmaloney): Move to an RAII wrapper.
     check_call(['rm', '-rf', install_dir])
 
-    print("Building package tarball")
+    with logger.scope("Build package tarball"):
+        print("Building package tarball")
 
-    # Check for forbidden services before packaging the tarball:
-    try:
-        check_forbidden_services(cache_abs("result"), RESERVED_UNIT_NAMES)
-    except ValidationError as ex:
-        raise BuildError("Package validation failed: {}".format(ex))
+        # Check for forbidden services before packaging the tarball:
+        try:
+            check_forbidden_services(cache_abs("result"), RESERVED_UNIT_NAMES)
+        except ValidationError as ex:
+            raise BuildError("Package validation failed: {}".format(ex))
 
-    # TODO(cmaloney): Updating / filling last_build should be moved out of
-    # the build function.
-    write_string(package_store.get_last_build_filename(name, variant), str(pkg_id))
+        # TODO(cmaloney): Updating / filling last_build should be moved out of
+        # the build function.
+        write_string(package_store.get_last_build_filename(name, variant), str(pkg_id))
 
-    # Bundle the artifacts into the pkgpanda package
-    tmp_name = pkg_path + "-tmp.tar.xz"
-    make_tar(tmp_name, cache_abs("result"))
-    os.rename(tmp_name, pkg_path)
-    print("Package built.")
+        # Bundle the artifacts into the pkgpanda package
+        tmp_name = pkg_path + "-tmp.tar.xz"
+        make_tar(tmp_name, cache_abs("result"))
+        os.rename(tmp_name, pkg_path)
+        print("Package built.")
```

Comment:
this shouldn't be inside the "building tarball" block. It's stating that the package build overall reached the end (do need to keep it as a print, provides feedback vs. the "building package" scope just ending).